tutorials/quickstart/Solution/Sales/Program.cs

```diff
@@ -26,6 +26,14 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                            endpointConfiguration.AuditProcessedMessagesTo("audit");
                            endpointConfiguration.SendHeartbeatTo("Particular.ServiceControl");
 
+                           var recoverablility = endpointConfiguration.Recoverability();
```

Comment:
```suggestion
                           // So that when we test recoverability, we don't have to wait so long
                           // for the failed message to be sent to the error queue
                           var recoverablility = endpointConfiguration.Recoverability();
```